Assets/PatchKit Patcher/Scripts/AppUpdater/AppUpdater.cs

```diff
@@ -93,7 +93,7 @@ AppContentSummary latestVersionContentSummary
             if (repairCost < contentSize)
             {
                 DebugLogger.Log(string.Format("Repair cost {0} is smaller than content cost {1}, repairing...", repairCost, contentSize));
-                IAppUpdaterStrategy repairStrategy = _strategyResolver.Create(StrategyType.RepairAndDiff, Context);
+                IAppUpdaterStrategy repairStrategy = _strategyResolver.Create(StrategyType.Repair, Context);
```

Comment:
Is there any point where `RepairAndDiff` can be triggered?

Reply:
Yes, pre update integrity checking never checks hashes. But pre-diff integrity checking can check hashes under certain conditions. So there's a possibility that the RepairAndDiff will be triggered.